lnrpc/invoicesrpc/invoices_server.go

```diff
@@ -265,7 +267,9 @@ func (s *Server) SubscribeSingleInvoice(req *SubscribeSingleInvoiceRequest,
 			}
 
 		case <-updateStream.Context().Done():
-			return updateStream.Context().Err()
+			return fmt.Errorf("subscription for "+
+				"invoice(pay_hash=%v):%v", hash,
```

Comment:
wrap error so that we can check `context.Cancel` further up the call stack?

Reply:
changed!

Reply:
nit: add space before `%w`.

Reply:
done!